features/features.go

```diff
@@ -0,0 +1,17 @@
+//go:build go1.16
+// +build go1.16
+
+package features
```

Comment:
issue: I try to avoid creating separate packages like this. I don't think it brings any value here. Please take a look at the usage of the API.

```go
suite := NewSuite(t, WithFeaturesFS(features.Features(), "example.feature"))
```

IMO it'd better make it simpler like the following

```go
suite := NewSuite(t, WithFeaturesFS(Features(), "example.feature"))
```

Reply:
I managed to change it, but for the record: it was only necessary for testing.